examples/quick-start-tail-sampling/README.md

```diff
@@ -0,0 +1,123 @@
+# Quick Start - Trace-based tests with Tail Sampling configuration
+
+> [Read the detailed recipe for setting up OpenTelemetry Collector with Tractest in our documentation.](https://docs.tracetest.io/examples-tutorials/recipes/running-tracetest-without-a-trace-data-store)
+
+This is a simple quick start example on how to set up `tail_sampling` into OTel Collector, allowing Tracetest to run tests in environments where we have a probabilistic sampler enabled and a percentage of the traces are not sent to the final data store.
+
+## Scenario
+
+In this scenario, we have on [Go API](./simple-go-service/) that sends Trace data to one instance of [OTel Collector](https://opentelemetry.io/docs/collector/), which samples 50% of the traces and sends it to [Jaeger](https://www.jaegertracing.io/) data store.
+
+```mermaid
+---
+title: Initial observability architecture
+---
+flowchart LR
+    GoAPI["Go API"]
+    OTelCol["OTel Collector"]
+    DataStore[("Jaeger")]
+
+    GoAPI -- Send traces --> OTelCol
+    OTelCol -- Forward 50% of Traces --> DataStore
+``````
+
+The collector configuration is defined by a `traces/jaeger` pipeline with one receiver, two processors (one to send traces in batch and the other set up the probabilistic sampling) and one exporter to Jaeger:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+processors:
+  batch:
+    timeout: 100ms
+  probabilistic_sampler:
+    hash_seed: 22
+    sampling_percentage: 50.0 # 50%
+
+exporters:
+  otlp/jaeger:
+    endpoint: jaeger:4317 # Send traces to Jager
+    tls:
+      insecure: true
+
+service:
+  pipelines:
+    traces/jaeger:
+      receivers: [otlp]
+      processors: [batch, probabilistic_sampler]
+      exporters: [otlp/jaeger]
+
+```
+
+However, to run a Trace-based test on this scenario and validate if the API is ok, we need to have the testing trace collected by our OTelCollector, how can we do that?
+
+Since Tracetest always starts a trace by adding a `tracetest=true` key/value to [TraceState](https://opentelemetry.io/docs/specs/otel/trace/api/#tracestate), that is [propagated](https://opentelemetry.io/docs/instrumentation/js/propagation/), we can filter these traces on the collector and send them to Tracetest.
```

Comment:
I would add a side note near this paragraph about the need for updated open telemetry libraries because `tracestate` is relatively new and not all languages SDKs were supporting it last year.

Example, the Javascript SDK only started supporting the `tracestate` field early this year.

Reply:
I have not added it because I want to check the SDK states and see the feedback from OTel team first.